engine/src/flutter/tools/gn

```diff
@@ -1187,6 +1189,13 @@ def parse_args(args):
       '--no-dart-dynamic-modules', dest='dart-dynamic-modules', action='store_false'
   )
 
+  parser.add_argument(
+      '--no-dart-secure-socket',
+      default=False,
+      action='store_true',
+      help='Disable secure socket support in Dart.',
```

Comment:
Can we add a clear message that this is a dangerous flag?

Reply:
done